ide/src/tests/runTests.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path';
```

Comment:
Since the tests are split across so many files, it would be good to have a document summarizing the test architecture. This could either be a comment at the top of this file, or a standalone `src/tests/README.md`. Just a paragraph or two explaining the directory structure and how data flows between each component.